pkg/errors/errors.go

```diff
@@ -193,6 +193,9 @@ var (
 
 	// ErrTimeout is raised when an operation has timed out
 	ErrTimeout = fmt.Errorf("operation timed out")
+
+	// ErrEndpointsNotAvailable is raised when no endpoints are available for the application
+	ErrEndpointsNotAvailable = "There are no available endpoints for '%s'.\n   Follow this link to know more about how to create public endpoints for your application:\n   https://www.okteto.com/docs/cloud/ssl/"
```

Comment:
As this is not an error itself, could we move this as the content of the message at the log? 

Not having endpoints its not an error necessarily.

Reply:
Resolved!